google/cloud/pubsub/publisher_connection_test.cc

```diff
@@ -153,7 +155,9 @@ TEST(PublisherConnectionTest, FlowControl) {
   EXPECT_THAT(rejected.get(), StatusIs(StatusCode::kFailedPrecondition));
 
   publisher->Flush({});
-  publish.PopFront().set_value();
+  for (int i = 0; i < publish_count; i++) {
+    publish.PopFront().set_value();
+  }
```

Comment:
Perhaps there should be an `AsyncSequencer::empty()`, removing the need to keep its size externally?

Reply:
Oops ... I see you mentioned that explicitly in the description.